src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1

```diff
@@ -213,11 +222,12 @@ PrivateData = @{
 
         # ReleaseNotes of this module
         ReleaseNotes = '* Fix Get-AzureRmVM -ResourceGroupName <rg> to return more than 50 results if needed
-* Added an example of the ''SimpleParameterSet'' to New-AzureRmVmss cmdlet help.
-* Fixed a typo in the Azure Disk Encryption progress message'
+        * Added an example of the ''SimpleParameterSet'' to New-AzureRmVmss cmdlet help.
+        * Fixed a typo in the Azure Disk Encryption progress message'
+        * ReleaseNotes = '* Shared Gallery cmdlets'
```

Comment:
I think  '* ReleaseNotes =' is duplicated.  Please remove it and combine * Shared Gallery cmdlets with the above ReleaseNotes.